cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -385,7 +385,9 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
           resultTable()
         }
 
-        p("All of the above mentioned privileges can be granted via the `ROLE MANAGEMENT` privilege. The following query shows an example of this:")
+        p(
+          """All of the above mentioned privileges, creating, deleting, assigning, removing and listing roles, can be granted via the `ROLE MANAGEMENT` privilege.
```

Comment:
```suggestion
          """All of the above mentioned privileges: creating, deleting, assigning, removing and listing roles; can be granted via the `ROLE MANAGEMENT` privilege.
```
My grammar isn't amazing so we should probably get docs help on this, but you're using a comma to separate lists and clauses together and it's a bit confusing. I think this is better, but it may still be wrong. This is probably the same everywhere.

Reply:
probably yes, not sure what we do on the places I didn't need to update (I know some just have a list instead of in text when it's a lot, but not sure about the smaller examples)